apps/coscrad-frontend/src/store/slices/id-generation/thunks/acquire-id.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpStatusCode, IHttpErrorInfo } from '@coscrad/api-interfaces';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { getConfig } from '../../../../config';
+import { buildAuthenticationHeaders } from '../../utils/build-authentication-headers';
+import { ID_GENERATION } from '../constants';
+
+export const acquireId = createAsyncThunk(`${ID_GENERATION}/ACQUIRE_ID`, async (_, thunkApi) => {
+    const { getState } = thunkApi;
+
+    const token = getState()['auth']?.userAuthInfo?.token;
```

Comment:
Given that we are using magic strings here to avoid circularities, I wonder if we could create a unit test for this logic? It seems risky.

Reply:
[Here](https://www.pivotaltracker.com/story/show/184107065) is a story for that.